README.md

```diff
@@ -19,3 +19,30 @@ quantitatively reason with them.
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+# Solution
+
+As seen on the slides, the probability of choosing a good (first-element) pivot is 1/2 or 50%.
+
+With the median-of-three strategy, we assume that all three elements (first, middle, and last) are independent and randomly distributed. For three elements there are 6 total possible arrangments:
+
+First, middle, last
+
+Middle, last, first
+
+Last, first, middle
+
+First,  last, middle
+
+Middle, first, last
+
+Last, middle, first
+
+This meanst that there are only two arrangments where the middle is near the actual median, meaning a good pivot choice (Last, middle, first and First, middle, last). This leads to a simple probability of $\frac{2}{6}=\frac{1}{3}$. Comparing this to a first-element pivot:
+
+$\frac{1}{2} ? \frac{1}{3} \rightarrow \frac{1}{2} > \frac{1}{3} \therefore \text{First element method is better than median-of-three}$
```

Comment:
That seems weird -- you're ruling out more bad pivots, but somehow the probability of getting a good pivot is less? Did you account for the different probabilities of the cases you list above?